### ISSUE TITLE
Try to generate plots like KPMP

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/portal-ui#1472]
@ngehlenborg : The request here is just to locate the necessary data and assemble a plot like these by hand, right? You're not asking for a new portal feature at this point?

<details>

![thumbnail_IMG_1528](https://user-images.githubusercontent.com/730388/105104749-e3da9980-5a80-11eb-8a97-dd210cd39a06.png)
![thumbnail_IMG_1527](https://user-images.githubusercontent.com/730388/105104756-e6d58a00-5a80-11eb-8ea3-4e69e7902c36.png)
![thumbnail_IMG_1526](https://user-images.githubusercontent.com/730388/105104762-ea691100-5a80-11eb-9bf4-a2b96028e172.png)

</details>

**Comment by ngehlenborg** [comment]
Yes, for now we only want to know what is possible with the data that we have. In the future, we might implement these kind of plots as a portal feature.
